fw/http_limits.c

```diff
@@ -1298,18 +1299,20 @@ frang_resp_process(TfwHttpResp *resp)
 			cli_addr = &req->conn->peer->addr;
 	}
 
+	if (unlikely(resp->content_length > body_len))
+		exceeded = resp->content_length;
+	else if (unlikely(resp->body.len > body_len))
+		exceeded = resp->body.len;
```

Comment:
Not a big deal, but according to our and the Linux kernel coding style there should be braces. Did you enable the github hook for checkpatch.pl?